src/bundle/Resources/public/js/scripts/admin.location.bookmark.js

```diff
@@ -1,6 +1,28 @@
 (function (global, doc) {
     const SELECTOR_FORM = 'form[name="location_update_bookmark"]';
-    const updateBookmark = () => doc.querySelector(SELECTOR_FORM).submit();
+    const SELECTOR_BOOKMARK_CHECKBOX = '#location_update_bookmark_bookmarked';
+    const SELECTOR_BOOKMARK_LOCATION_INPUT = '#location_update_bookmark_location';
 
-    doc.querySelector('#location_update_bookmark_bookmarked').addEventListener('change', updateBookmark, false);
+    const submitBookmarkForm = () => doc.querySelector(SELECTOR_FORM).submit();
+    const updateBookmarkCheckbox = (bookmarked) => {
+        const checkbox = doc.querySelector(SELECTOR_BOOKMARK_CHECKBOX);
+
+        checkbox.checked = bookmarked;
+    }
```

Comment:
CS: Missing semicolon. Please, run Prettier to fix these issues.

Reply:
As far as I know, we don't have Prettier in admin-ui configured.